py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -63,7 +84,9 @@ def create_event(self) -> dict:
         ingestion lambda
         """
         return {
-            "files": self.filenames,
+            "prefix": self.prefix,
+            "suffix": self.suffix,
+            "filenames": self.filenames,
```

Comment:
event is now these three items, which can be uncompressed into full file names for ingestion.